# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome2/acme/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


Oh my god is Gnome full of nice code duplications. This is exactly the same
patch as gnome-terminal needs (yes applies cleanls ;-): Some real bugs
uncovered by gcc-3.4 ...

  - Rene Rebe <rene@exactcode.de>

--- gnome-terminal-2.6.0/src/eggcellrendererkeys.c.orig	2004-05-10 13:41:45.587354088 +0200
+++ gnome-terminal-2.6.0/src/eggcellrendererkeys.c	2004-05-10 13:42:16.827604848 +0200
@@ -584,7 +584,7 @@
 void
 egg_cell_renderer_keys_set_accelerator (EggCellRendererKeys *keys,
                                         guint                keyval,
-                                        GdkModifierType      mask)
+                                        EggVirtualModifierType mask)
 {
   char *text;
   gboolean changed;
@@ -626,7 +626,7 @@
 void
 egg_cell_renderer_keys_get_accelerator (EggCellRendererKeys *keys,
                                         guint               *keyval,
-                                        GdkModifierType     *mask)
+                                        EggVirtualModifierType *mask)
 {
   g_return_if_fail (EGG_IS_CELL_RENDERER_KEYS (keys));
 
